backend/authentication/backend.py

```diff
@@ -33,11 +33,11 @@ async def authenticate(
         self, request: Request
     ) -> t.Optional[tuple[authentication.AuthCredentials, authentication.BaseUser]]:
         """Handles JWT authentication process."""
-        if "Authorization" not in request.headers:
+        cookie = request.cookies.get("BackendToken")
```

Comment:
We can probably drop the `Backend` here, maybe a bit pedantic though.
```suggestion
        cookie = request.cookies.get("token")
```